Support for swapping into LBC

infinite-persistence opened the pull request:
## Issue
- Closes #5503: [Add support for swapping into LBC](https://github.com/lbryio/lbry-desktop/issues/5503)
- Requires https://github.com/lbryio/lbry-redux/pull/389

## Status (2021-04-09)
- Good to go. Some more testing + actual alt coin swaps would be good.

kauffj commented:
@tiger5226 is adding an endpoint for swap caps.

seanyesmunt reviewed (commented):
a couple comments after a quick look. I will look closer when I can test it better.

infinite-persistence commented:
### Update
- Rebased to latest master
- Changed `btcAddresses` to `btc_addresses` to match the rest
- "Bitcoin":  use `FormField`'s label instead of another div.
